changelog.d/20240422_145951_salome.voltz_scrt_4442_ggshield_secret_scan_repo_ignores_exclusion_patterns.md

```diff
@@ -0,0 +1,3 @@
+### Fixed
+
+The option `--exclude PATTERN` is no longer ignored by the command `ggshield secret scan repo`.
```

Comment:
nit: content of the changelog entry must be a list, even if there is only one item, otherwise the generated changelog looks odd:

```suggestion
- The option `--exclude PATTERN` is no longer ignored by the command `ggshield secret scan repo`.
```